projects/scion/microfrontend-platform/src/lib/testing/spec.util.spec.ts

```diff
@@ -49,7 +48,8 @@ export function expectPromise(actual: Promise<any>): PromiseMatcher {
         fail(`Promise expected to be rejected but was resolved: ${value}`);
       }
       catch (reason) {
-        const expectedErrorMessage = expected && (typeof reason === 'string' ? reason : (reason instanceof Error ? reason.message : reason.toString()));
+        const errorMessage = reason instanceof Error ? reason.message : reason.toString();
+        const expectedErrorMessage = expected && errorMessage;
```

Comment:
You can use `stringifyError`, as follows:
```ts
const expectedErrorMessage = expected && stringifyError(reason );
```

or even change it as follows:
```ts
if (expected && !stringifyError(reason).match(expected)) {
  fail(`Expected promise to be rejected with a reason matching '${expected.source}', but was '${stringifyError(reason)}'.`);
}
else {
  expect(true).toBeTrue();
}
```

Reply:
Done. I used the second suggestion.